pkgs/development/python-modules/pysaml2/default.nix

```diff
@@ -27,6 +28,11 @@ buildPythonPackage rec {
       src = ./hardcode-xmlsec1-path.patch;
       inherit xmlsec;
     })
+    (fetchpatch {
```

Comment:
do you mind adding a comment saying it's needed to be remove next release as it's in master
```suggestion
    # remove on next release
    (fetchpatch {
```

Reply:
Yes of course - done.